src/oca_github_bot/process.py

```diff
@@ -12,7 +12,9 @@ def call(cmd, cwd):
     return subprocess.call(cmd, cwd=cwd)
 
 
-def check_call(cmd, cwd, log_error=True):
+def check_call(cmd, cwd, log_error=True, extra_cmd_args=False):
```

Comment:
```suggestion
def check_call(cmd, cwd, log_error=True, extra_cmd_args=None):
```

False for this usage is an Odooism :)

Reply:
;-)